lib/alexa.js

```diff
@@ -367,6 +367,9 @@ function ResponseBuilder(self) {
 
                 responseObject.response.directives = [audioPlayerDirective];
                 return this;
+            },
+            'setResponse': function () {
```

Comment:
Sure, since you have already put an effort in updating doc, why not get them in. :)